build.sbt

```diff
@@ -7,10 +7,21 @@ import sbt.Keys._
 import sbt.librarymanagement.ModuleID
 import sbt.nio.Keys._
 
-lazy val scala212 = "2.12.8"
-lazy val scala211 = "2.11.12"
+lazy val scalaNew = "2.12.8"
```

Comment:
nit: I think it's better to keep descriptive names like 212 and 211. "new" and "old" tend to not age well.

Reply:
reverted